tests/suite/test_dos.py

```diff
@@ -449,3 +453,107 @@ def test_dos_arbitrator(
         assert (
                 len(learning_units_hostname) == 2
         )
+
+    @pytest.mark.skip
```

Comment:
Did you mean to leave the `skip` here?

Reply:
The skip added by Sean O'Neill, I don't know for sure why it changed to skip.

Reply:
What do you mean added by Sean?
If you mark a test with `skip` it will not run

Reply:
I think there were concerns about creating an actual DoS attack since these tests would run in GKE. Do you think it would be a problem for these specific tests?

I also feel like we should find an alternative solution, adding automation tests but marking them with `skip` is like not adding them at all, since they will never run.

cc @ciarams87

Reply:
I run those tests in my minikube environment and they are all passed.
If there is no problem to rum them on GKE, need to run them there.
otherwise, need to understand why we cannot run those tests and try to solve the problems.
Case we cannot solve the problems and rum the tests in GKE, maybe we can run them on minikube.

Reply:
I think that if they can be run in `minikube`/`kind` successfully, then there's no need to run them in GKE